packages/twenty-server/src/engine/metadata-modules/workspace-migration/workspace-migration.entity.ts

```diff
@@ -14,6 +14,7 @@ export enum WorkspaceMigrationColumnActionType {
   DROP_FOREIGN_KEY = 'DROP_FOREIGN_KEY',
   DROP = 'DROP',
   CREATE_COMMENT = 'CREATE_COMMENT',
+  CREATE_FOREIGN_TABLE = 'CREATE_FOREIGN_TABLE',
```

Comment:
I am not sure we should put that into the WorkspaceMigrationColumnActionType, does not look like a column migration action.

I think this enum should be WorkspaceMigrationAlterTableActionType
but even in this case, we should not put the createForeignTable here

Reply:
good catch, forgot to remove it. This is not used